### PR TITLE
Specify Agent Name

### PR DESCRIPTION
**Pull request by nhmathis**
- Update to specify agent name
- Persist agent on multiple docker-compose ups
- Critical for saving new demo images and maintain UCD configuration

### CODE REVIEW

File: compose/Deploy/docker-compose.yaml [hunk 1]
```diff
@@ -20,6 +20,8 @@ services:
       - 7918:7918
```
Comment (adamkingit) [inline compose/Deploy/docker-compose.yaml]: We should probably remove the ^Ms from this file. Sorry we let them leak into it in the first place.

Comment (nhmathis) [reply compose/Deploy/docker-compose.yaml]: Can do

Comment (nhmathis) [reply compose/Deploy/docker-compose.yaml]: https://help.github.com/articles/dealing-with-line-endings/ 

GitHub's solution

**Comment by nhmathis** [comment]
-`^M`s have been removed
-If `AGENT_NAME` is not specified, the original functionality will be used. Specifying AGENT_NAME will allow reuse of agents for UCD configurations.

**Pull request merged**